src/OpenFeature/OpenFeature.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0;net8.0;net462</TargetFrameworks>
```

Comment:
@askpt sorry, I didn't notice this before... this is removing `netstandard2.0`, which IIUC means we won't have support for some older platforms, specifically I'm thinking about support for full .NET framework 4.x.x. Am I correct on this? If so, is that really what we want?

Reply:
Hey @toddbaert! I blame the tools! 

The `netstandard2.0` TFM is still there. It just moved to 3rd instead of 1st.

Reply:
:facepalm: Ha! I'm sorry. Disregard then. I'm merging this.